src/main/java/org/sagebionetworks/web/client/presenter/DownPresenter.java

```diff
@@ -62,15 +58,12 @@ public void onSuccess(StackStatus status) {
 						break;
 					case READ_ONLY:
 					case DOWN:
-						// it's down, report the message and check again later
-						view.setMessage(status.getCurrentMessage());

```

Comment:
Alternatively, we could try to surface the status message to the user in the react component.
Perhaps we pass a string into _createSRCDown() and use it for the message.  Called once during an attach event (with the string "We're busy updating Synapse for you and will be back soon."), and in setMessage().

Reply:
Sure, I will work on that. But I will merge this one first.

Reply:
Actually, looking at the design's figma file, the message has 2 parts...a title (e.g. Sorry, Synapse is down for maintenance.) and another message (e.g. We’re busy updating Synapse for you and will be back soon). See https://sagebionetworks.jira.com/browse/SWC-5373. The original setMessage() seems to only provide the title part.

Reply:
I hardcoded the title but the smaller more information paragraph will be the error message from the backend.